maxim/logger/livekit/agent_activity.py

```
@@ -308,18 +318,111 @@ def handle_agent_speech_finished(self):
+                    name="User Audio Input",
+                    timestamp=int(time.time()),
+                ),
+            )
+        turn.turn_input_audio_buffer = None
+        get_session_store().set_session(session_info)
+    except Exception as e:
+        scribe().warning(
+            f"[Internal][{self.__class__.__name__}] handle_end_of_speech failed in function; error={e!s}\n{traceback.format_exc()}"
+        )
 
 def pre_hook(self, hook_name, args, kwargs):
```

Ruff (0.12.2)

`pre_hook` is too complex (18 > 10)

(C901)